src/main/java/net/openhft/chronicle/core/util/Longs.java

```diff
@@ -33,7 +33,6 @@
  * A collection of functional compositions to check and assert various requirements
  * related to {@code long} values.
  */
-@Deprecated(/* to be moved to internal in Chronicle-Bytes in x.26 as that is where it's used. One method is used in UnsafeMemory */)
```

Comment:
if we are going to de-deprecate classes/methods then this change should also be made in another PR and merged ASAP into develop. That way, people won't remove calls to methods which are no longer deprecated

Reply:
done